Tests iOS/ReaderTests.swift

```diff
@@ -98,17 +98,23 @@ class ReaderTests: XCTestCase {
         moveFromArchiveToSavesEvent!.getContentContext()!.assertHas(url: "http://example.com/items/archived-item-1")
     }
 
-    func test_tappingOverflowMenu_showsOverflowOptions() {
+    @MainActor
+    func test_tappingOverflowMenu_showsOverflowOptions() async {
         launchApp_andOpenItem()
         openReaderOverflowMenu()
         XCTAssertTrue(app.readerView.displaySettingsButton.exists)
         XCTAssertTrue(app.readerView.favoriteButton.exists)
         XCTAssertTrue(app.readerView.addTagsButton.exists)
         XCTAssertTrue(app.readerView.deleteButton.exists)
         XCTAssertTrue(app.readerView.shareButton.exists)
+
+        let overflowEvent = await snowplowMicro.getFirstEvent(with: "reader.toolbar.overflow")
+        overflowEvent!.getUIContext()!.assertHas(type: "button")
+        overflowEvent!.getContentContext()!.assertHas(url: "http://localhost:8080/hello")
     }
 
-    func test_tappingDisplaySettings_showsDisplaySettings() {
+    @MainActor
+    func test_tappingDisplaySettings_showsDisplaySettings() async {
```

Comment:
this test fails so I didn't not add it back to the suite, added the analytics which passes if I comment out the fontSelection / fontSize adjustment code